knowledge-base/drawer-as-side-navigation.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This could be linked from the Navigation article of the drawer too

Reply:
It is already linked.